play.py

```diff
@@ -0,0 +1,21 @@
+import player
+import game
+import argparse
+
+parser = argparse.ArgumentParser(description="setup tictactoe game")
+parser.add_argument('b', nargs='?', default=3, type=int, help="board size")
+parser.add_argument('p1', nargs='?', default="human", type=str, help="player2 human or computer")
+parser.add_argument('pi1', nargs='?', default="X", type=str, help="player1 piece shape")
+parser.add_argument('p2', nargs='?', default="computer", type=str, help="player2 human or computer")
+parser.add_argument('pi2', nargs='?', default="O", type=str, help="player2 piece shape")
```

Comment:
Ditto here for all these args; they should all be optional IMO, with sane defaults for a standard bpard size and a human vs comp game.